Mathlib/CategoryTheory/Monoidal/Braided/Basic.lean

```diff
@@ -504,18 +502,20 @@ end CommMonoid
 
 section Tensor
 
+variable {C}
+
 /-- The strength of the tensor product functor from `C × C` to `C`. -/
```

Comment:
I believe that this docstring should be edited. Otherwise, this looks good to me.